src/views/Settings.vue

```diff
@@ -57,11 +57,13 @@ export default defineComponent({
   },
   methods: {
     setFacility (facility: any) {
-      this.userProfile.facilities.map((fac: any) => {
+      if(this.userProfile && this.userProfile.facilities) {
+        this.userProfile.facilities.map((fac: any) => {
         if (fac.facilityId == facility['detail'].value) {
```

Comment:
Improve indentation

Reply:
Done Sir.
Thank You Sir for reviewing my code.